App.js

```diff
@@ -18,7 +19,7 @@ export default class App extends Component {
       <View style={styles.container}>
         <Header />
         {
-          currentPage === 'Home' ? <Home /> : null
+          currentPage === 'Home' ? <Home /> : <Login />
```

Comment:
We're going to need a way to have logic for our 4-5 different pages. So for now we can use a single ternary to go through all of them while we're making them.
currentPage === 'Home' ? <Home /> : currentPage === 'Login' ? <Login /> : null 
etc